### DISCUSSION TITLE
Increase priority for public transit stop names over retail

### DISCUSSION THREAD

**Discussion by dickorydock** [organicmaps/organicmaps#7116] [Ideas]
In an effort to improve the Organic Maps experience for public transit riders, I'd like to suggest that public transit stops and names be given higher priority than nearby shops. I'm attaching two screenshots from New Orleans where you can clearly see that shops and restaurants appear but the streetcar stop "Dumaine Street" doesn't show up until you zoom in further. This makes it harder for people to see where they are in relation to transit, making Organic Maps more difficult to use for public transit riders. I realize that some transit systems (like buses) have very frequent stops, and I'm not saying all of those should be labelled, though an icon would be helpful. In the screenshots, there is neither an icon nor a name until you zoom in further, even as multiple nearby businesses are included on the map. 

![IMG_4683](https://github.com/organicmaps/organicmaps/assets/13089612/4d2e7e55-0c2e-4bf2-b526-6309ce807526)
![IMG_4684](https://github.com/organicmaps/organicmaps/assets/13089612/be94c0a9-614f-4b49-ae30-e213e2a5b71d)

**Comment by pastk** [comment]
Thanks for your feedback!
Actually tram stops have a quite high priority (higher than most of the shops), but in this particular case the stop is being displaced by a ["Shops of The Colonnade" mall](https://www.openstreetmap.org/relation/7064857) and shopping malls have even higher priority because often they're big and are landmarks by themselves.

It'd make sense to increase PT features priorities in a "Public Transport" map style, see #1577
